test/v2/models/response/retrieveHistoricNonFhlUkPropertyAnnualSubmission/AnnualAdjustmentsSpec.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.response.retrieveHistoricNonFhlUkPropertyAnnualSubmission
+
+import play.api.libs.json.{JsValue, Json}
+import support.UnitSpec
+import v2.models.response.retrieveHistoricNonFhlUkPropertyAnnualSubmissionResponse.{AnnualAdjustments, RentARoom}
+
+
+class AnnualAdjustmentsSpec extends UnitSpec{
+
+  private def decimal(value: String): Option[BigDecimal] = Option(BigDecimal(value))
+
+  val downstreamJson: JsValue = Json.parse("""
+                                             |   {
+                                             |      "lossBroughtForward": 200.00,
+                                             |      "balancingCharge": 200.00,
+                                             |      "privateUseAdjustment": 200.00,
+                                             |      "businessPremisesRenovationAllowanceBalancingCharges": 80.02,
+                                             |      "nonResidentLandlord": true,
+                                             |      "ukRentARoom": {
+                                             |         "jointlyLet": true
+                                             |      }
+                                             |   }
+                                             |""".stripMargin)
+  val mtdJson: JsValue = Json.parse("""
+                                             |   {
+                                             |      "lossBroughtForward": 200.00,
+                                             |      "balancingCharge": 200.00,
+                                             |      "privateUseAdjustment": 200.00,
+                                             |      "businessPremisesRenovationAllowanceBalancingCharges": 80.02,
+                                             |      "nonResidentLandlord": true,
+                                             |      "rentARoom": {
+                                             |         "jointlyLet": true
+                                             |      }
+                                             |   }
+                                             |""".stripMargin)
+
+  val model: AnnualAdjustments = AnnualAdjustments(
+    lossBroughtForward = decimal("200.00"),
+    privateUseAdjustment = decimal("200.00"),
```

Comment:
Can you change the numbers from 200, 200, and 200 to 200, 300, 400 so that you can see that the correct data is being delivered to the correct place? When running the tests I did this and found the test failed as the ordering was correct. Should be a simple fix - just ordering and then making sure the test passes

Reply:
Doesn't matter about the values and the order as long as you explicitly declare what parameters is assigned to their values.